### PR TITLE
DOCS-1798: Add performance metrics capture

### PR DESCRIPTION
**Pull request by npentrel**
Hey @ale7714 and @MirandaKapin, I'm thinking of adding this page to make it especially easy to use the new module: 
https://docs-test.viam.dev/2607/data/capture/performance-metrics/

### CODE REVIEW

File: docs/data/capture/performance-metrics.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+# SME: Alexa Greenberg
+---
+
+You can capture and monitor the following metrics about the performance of individual machines or your entire fleet:
+
+- **Wireless Signal Strength and Quality**: signal level, link quality, and noise level
+- **Memory Usage**: memory statistics, including total available memory, used percentage, and specifics on various types of memory (cached, free, slab, etc.)
+- **CPU Utilization**: CPU usage across different states (user, system, idle, etc.)
+- **Disk I/O**: Metrics on read and write operations, including bytes transferred and operation times
+- **Network Traffic**: Detailed network statistics, including bytes sent and received, packet information, and error counts, provide a deep dive into a device's network performance.
```
Comment (npentrel) [inline docs/data/capture/performance-metrics.md:21]:
```suggestion
- **Wireless Signal Strength and Quality**: signal level, link quality, and noise level
- **Memory Usage**: memory statistics, including total available memory, used percentage, and specifics on various types of memory (cached, free, slab, etc.)
- **CPU Utilization**: CPU usage across different states (user, system, idle, etc.)
- **Disk I/O**: metrics on read and write operations, including bytes transferred and operation times
- **Network Traffic**: detailed network statistics, including bytes sent and received, packet information, and error counts, provide a deep dive into a device's network performance.
```

File: docs/data/capture/performance-metrics.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+# SME: Alexa Greenberg
+---
+
+You can capture and monitor the following metrics about the performance of individual machines or your entire fleet:
+
+- **Wireless Signal Strength and Quality**: signal level, link quality, and noise level
+- **Memory Usage**: memory statistics, including total available memory, used percentage, and specifics on various types of memory (cached, free, slab, etc.)
+- **CPU Utilization**: CPU usage across different states (user, system, idle, etc.)
+- **Disk I/O**: metrics on read and write operations, including bytes transferred and operation times
+- **Network Traffic**: detailed network statistics, including bytes sent and received, packet information, and error counts, provide a deep dive into a device's network performance
```
Comment (sguequierre) [inline docs/data/capture/performance-metrics.md:22]:
```suggestion
- **Network Traffic**: detailed network statistics, including bytes sent and received, packet information, and error counts, providing a deep dive into a device's network performance
```

File: docs/data/capture/performance-metrics.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+
+## Requirements
+
+To capture data from a machine, you need to [create a machine in Viam](https://docs.viam.com/manage/fleet/machines/#add-a-new-machine) and follow the setup instruction to install `viam-server` and connect it to the Viam app.
```
Comment (sguequierre) [inline docs/data/capture/performance-metrics.md:26]:
```suggestion
To capture data from a machine, you need to [create a machine in Viam](https://docs.viam.com/manage/fleet/machines/#add-a-new-machine) and follow the setup instructions to install `viam-server` and connect it to the Viam app.
```
I think

File: docs/data/capture/performance-metrics.md [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+
+{{< alert title="Note" color="note" >}}
+You must run `viam-server` with `sudo` to monitor machine performance metrics.
+{{< /alert >}}
+
+## Add sensor to measure performance metrics
+
+To obtain performance metrics about your machine, use the [`viam-telegraf-sensor`](https://app.viam.com/module/viam/viam-telegraf-sensor) module from the [Viam registry](/registry/).
+The module provides a `sensor` that allows you to obtain readings containing your machine's performance metrics.
```
Comment (sguequierre) [inline docs/data/capture/performance-metrics.md:35]:
```suggestion
The module provides a [sensor](/components/sensor/) that allows you to obtain readings containing your machine's performance metrics.
```
(opt)

File: docs/data/capture/performance-metrics.md [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+
+1. Go to your machine's **Config** page and click **Create component**.
+2. Then select the `viam:viam-sensor:telegrafsensor` model from the [`viam-telegraf-sensor` module](https://app.viam.com/module/viam/viam-telegraf-sensor).
+3. Click **Add module**, then enter a name for your sensor, for example `my-telegrafsensor`, and click **Create**.
+
+In the next step you will configure the data manager to capture and sync the performance metrics from your configured sensor.
```
Comment (sguequierre) [inline docs/data/capture/performance-metrics.md:41]: (opt) I would veer on the side of referring to it as data management service consistently. Also not sure if this sentence is necessary although it does provide a bit of a transition. Maybe consider line 43 before 41?

File: docs/data/capture/performance-metrics.md [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+### Test sensor data
+
+After you configure your sensor, navigate to the [Control tab](/fleet/machines/#control) and select the **Sensors** dropdown panel.
+To access detailed readings from your sensor, click on the **Get Readings** button.
+
+{{<gif webm_src="/data/monitor.webm" mp4_src="/data/monitor.mp4" alt="sensor control tab">}}
+
+## Configure data manager to capture and sync sensor data
+
+To capture the data from your configured sensor, you need to add the data management service and configure it to capture and sync the sensor data:
```
Comment (sguequierre) [inline docs/data/capture/performance-metrics.md:54]:
```suggestion
To capture the data from your configured sensor, you need to add the [data management service](https://docs.viam.com/data/) and configure it to capture and sync the sensor data:
```
(opt)

File: docs/data/capture/performance-metrics.md [hunk 7]
```diff
@@ -0,0 +1,78 @@
...
+
+1. On your machine's **Config** page, go to the **Services** subtab and click **Create service**.
+2. Select the **data management** service and give it a name.
+   For example `data_manager`.
```
Comment (sguequierre) [inline docs/data/capture/performance-metrics.md:58]:
```suggestion
   For example, `data_manager`.
```
I think although maybe fine?

File: docs/data/capture/performance-metrics.md [hunk 8]
```diff
@@ -0,0 +1,78 @@
...
+3. Click **Create**.
+4. Go to the **Components** subtab and find your telegrafsensor.
```
Comment (sguequierre) [inline docs/data/capture/performance-metrics.md:60]:
```suggestion
4. Go to the **Components** subtab and find your `telegrafsensor`'s configuration card.
```
(opt)

File: docs/data/capture/performance-metrics.md [hunk 9]
```diff
@@ -0,0 +1,78 @@
...
+### View data
+
+View your sensor data on the [**Data** tab](https://app.viam.com/data/view?view=sensors).
+
+![View of sensor data](/data/sensor-data.png)
+
+### Next steps
+
+The data you obtain about your machines is associated with metadata about the machine and time of capture.
+Once you have captures data about your machines, you can query your captured data with any tools that with SQL or MQL or visualize your data with tools like Grafana:
```
Comment (sguequierre) [inline docs/data/capture/performance-metrics.md:73]:
```suggestion
Once you have captured data about your machines, you can query your captured data with any tools that with SQL or MQL or visualize your data with tools like Grafana:
```

**Review by sguequierre — CHANGES_REQUESTED** [review]
This is very cool-- some small fixes and suggestions otherwise LGTM!

File: docs/data/capture/performance-metrics.md [hunk 10]
```diff
@@ -0,0 +1,78 @@
...
+To obtain performance metrics about your machine, use the [`viam-telegraf-sensor`](https://app.viam.com/module/viam/viam-telegraf-sensor) module from the [Viam registry](/registry/).
+The module provides a [sensor](/components/sensor/) that allows you to obtain readings containing your machine's performance metrics.
+
+1. Go to your machine's **Config** page and click **Create component**.
+2. Then select the `viam:viam-sensor:telegrafsensor` model from the [`viam-telegraf-sensor` module](https://app.viam.com/module/viam/viam-telegraf-sensor).
+3. Click **Add module**, then enter a name for your sensor, for example `my-telegrafsensor`, and click **Create**.
+
+In the next step you will configure the data manager to capture and sync the performance metrics from your configured sensor.
+
+For more information, see [add a module from the Viam Registry](https://docs.viam.com/registry/configure/#add-a-modular-resource-from-the-viam-registry).
```
Comment (npentrel) [inline docs/data/capture/performance-metrics.md:43]:
```suggestion
For more information, see [add a module from the Viam Registry](https://docs.viam.com/registry/configure/#add-a-modular-resource-from-the-viam-registry).

In the next step you will configure the data management service to capture and sync the performance metrics from your configured sensor.
```

File: docs/data/capture/performance-metrics.md [hunk 11]
```diff
@@ -0,0 +1,78 @@
...
+
+### Test sensor data
+
+After you configure your sensor, navigate to the [Control tab](/fleet/machines/#control) and select the **Sensors** dropdown panel.
+To access detailed readings from your sensor, click on the **Get Readings** button.
+
+{{<gif webm_src="/data/monitor.webm" mp4_src="/data/monitor.mp4" alt="sensor control tab">}}
+
+## Configure data manager to capture and sync sensor data
```
Comment (npentrel) [inline docs/data/capture/performance-metrics.md:52]:
```suggestion
## Configure data management service to capture and sync sensor data
```

**Comment by npentrel** [comment]
@sguequierre ready for re-review

**Review by sguequierre — APPROVED** [review]

**Pull request merged**